frontend/src/components/CollectiveCard.js

```diff
@@ -29,11 +48,11 @@ export default class CollectiveCard extends Component {
             </div>
             <div className='CollectiveCard-body'>
               <div className='CollectiveCard-name'>{name}</div>
-              <div className='CollectiveCard-description'>{description}</div>
+              <div className='CollectiveCard-description'>We are on a mission to {description}</div>
```

Comment:
Now that this is not only for the homepage, we should move that to the `strings.json` and use `i18n`.

But it's tricky because 
- you should actually use the language of that group and not of that page when you render it.
- you should only show related collectives in the same language.